client-app/core/api/graphql/client.ts

```diff
@@ -1,41 +1,13 @@
-import { ApolloClient, InMemoryCache } from "@apollo/client/core";
-import { cache, deprecatedLink, link, options } from "@/core/api/graphql/config";
-
-const fetchPolicy = "no-cache";
-
-/**
- * Non-cached version of Apollo Client
- *
- * @deprecated
- *
- * Use {@link apolloClient} instead.
- *
- * **IMPORTANT!** Because of cache usage, query deduplication and `@vue/apollo-composable` usage,
- * you can't just replace {@link graphqlClient} with {@link apolloClient} in your code.
- * Look at https://github.com/VirtoCommerce/vc-theme-b2b-vue/pull/868 for more information.
- */
-export const graphqlClient = new ApolloClient({
-  ...options,
-  link: deprecatedLink,
```

Comment:
We can safely use `link` instead of `deprecatedLink`, because the first one differs by having `cartLink` and condition use  `wsLink` for subscriptions and `httpLink` for others.
Cart link has a defined list of operations it will be applied for. So it doesn't influence anything else.
The second also works perfectly well, because all subscriptions should use `wsLink` (actually we have only one which already use our "new" client.